server/repository/migrations/postgres/2023-03-09-194728_v1_1_5/up.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE name ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE changelog ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE clinician ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE clinician_store_join ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE name_store_join ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE document ADD is_sync_update BOOLEAN NOT NULL DEFAULT FALSE;
+
+/* View wasn't being dropped in down.sql, so have moved it here to recreate view */
```

Comment:
down.sql is not called anyway while doing the normal migrations, only if you explicitly reverse or redo a migration...